pkg/minikube/node/start.go

```diff
@@ -89,7 +89,10 @@ type Starter struct {
 // Start spins up a guest and starts the Kubernetes node.
 func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 	var kcs *kubeconfig.Settings
-	if starter.Node.KubernetesVersion == constants.NoKubernetesVersion { // do not bootstrap cluster if --no-kubernetes
+	// Do not bootstrap cluster if --no-kubernetes.
+	if viper.GetBool("no-kubernetes") ||
+		starter.Node.KubernetesVersion == constants.NoKubernetesVersion {
```

Comment:
lets make this code more readable

Reply:
Yes, I have removed this code, thanks for the note.